### ISSUE TITLE
Cannot select and copy error message text from left UI column

### ISSUE THREAD

**Issue by bahmutov** [cypress-io/cypress#344]
When there is an error, the left UI column shows a link to the docs, but I cannot select the link text.
Instead shows popup that the output has been printed to the console, but there is nothing in the console. It would be nice if I could just click on the link and open it

<img width="427" alt="screen shot 2016-12-14 at 11 04 08 am" src="https://cloud.githubusercontent.com/assets/2212006/21189644/5bc99fbc-c1ed-11e6-8e15-26678dd93432.png">

**Comment by bahmutov** [comment]
Also, one of the reasons for the remaining commands could be syntax error (like `ReferenceError`) in the test

```js
it('does not work', () => {
  cy.get(undefinedName).click()
})
```
The above code fails with "queue ended early" error which is hard to debug

**Comment by jennifer-mann** [comment]
[PR to fix this issue](https://github.com/cypress-io/cypress-core-reporter/pull/1).

**Comment by brian-mann** [comment]
Fixed in [`0.19.1`](https://github.com/cypress-io/cypress/wiki/Changelog#0191-03092017).
